Answers.md

```diff
@@ -2,10 +2,21 @@
 
 1. What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
-1. What does it mean to think in react?
+It is a Javascript Library used to create user interface and user experience. It breaks down a web site into small pieces, by making them into components. 
 
-1. Describe state.
+2. What does it mean to think in react?
 
-1. Describe props.
+The thought process a developer uses to break down the production of a project. Thinking through which components will be needed, what heirarchy to use, and how the file architecture should be, this is the meaning behind; "Thinking in react".
 
-1. What are side effects, and how do you sync effects in a React component to state or prop changes?
+3. Describe state.
+
+"State" in react means how a component is rendered and how it behaves if something acts on it through a function. It is dynamic and the components are interactive in React, for example; buttons, counters or text.
+
+
+4. Describe props.
+
+"Props" in react are the parameters used to control and customize different components. Props are connected to components and can be changed if necessary. Example; if we have an image, the url of that image is a prop and can be controlled to show different images if necessary.

```

Comment:
Great work! Keep in mind that the props keyword specifically allows us to pass variables and functions between components. You could use props to pass an image url if it was set to a variable though!